content/zh-cn/docs/concepts/policy/node-resource-managers.md

```diff
@@ -331,7 +331,7 @@ listed in alphabetical order:
 : Prevent all the pods regardless of their Quality of Service class to run on reserved CPUs
   (available since Kubernetes v1.32)
 
-`prefer-align-cpus-by-uncorecache` (alpha, hidden by default)
+`prefer-align-cpus-by-uncorecache` (beta, visible by default)
```

Comment:
It's required to change the zh text too:

```
`prefer-align-cpus-by-uncorecache`（Beta，默认可见）：
```

Reply:
Done, thanks